src/platform/bitboxbase/leds.c

```diff
@@ -15,33 +15,69 @@
 #include "leds.h"
 #include "bitboxbase_pins.h"
 
-static void _turn_small_led(int led, bool level)
+#include <stdint.h>
+static uint8_t small_leds[] = {PIN_LED_SMALL(0),
+                               PIN_LED_SMALL(1),
+                               PIN_LED_SMALL(2),
+                               PIN_LED_SMALL(3),
+                               PIN_LED_SMALL(4)};
+
+void turn_small_led(int led, bool level)
```

Comment:
Change `int led` to something like `enum leds_small_led_t`. You should then switch over the enumeration instead of directly passing the pin to `gpio_set_pin_level`.

Reply:
Passing the index of the LED (0 -> leftmost, 4 -> rightmost) is the best solution imho.